drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -45,7 +45,8 @@ function DragNDropTemplates(url_name) {
             'draggable': !item.drag_disabled,
             'aria-grabbed': item.grabbed,
             'data-value': item.value,
-            'data-drag-disabled': item.drag_disabled
+            'data-drag-disabled': item.drag_disabled,
+            'aria-describedby': gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.'),
```

Comment:
@cptvitamin What's the advantage of doing it that way, versus just making an sr div for the describedby as needed (i.e. only put the "Correctly placed" message in if the item has been placed)?